salt/metalk8s/addons/dex/deployed/chart.sls

```diff
@@ -55,10 +50,10 @@ metadata:
     app.kubernetes.io/managed-by: salt
     app.kubernetes.io/name: dex
     app.kubernetes.io/part-of: metalk8s
-    app.kubernetes.io/version: 2.30.0
-    helm.sh/chart: dex-0.6.3
+    app.kubernetes.io/version: 2.31.1
+    helm.sh/chart: dex-0.8.2
     heritage: metalk8s
-  name: dex
```

Comment:
We should remove this one in post upgrade then (and remove `dex-cluster` in post downgrade)